scss/forms/_quantity-selector.scss

```diff
@@ -31,7 +31,15 @@
       @include button-icon($quantity-selector-icon-remove, $size: $quantity-selector-icon-width $quantity-selector-icon-remove-height, $pseudo: "after");
       order: -1;
       padding-right: $quantity-selector-btn-padding-x;
-      border-right: none;
+      border-right-width: 0;
+
+      &:focus {
+        &[data-focus-visible-added] { // stylelint-disable-line selector-no-qualifying-type
+          padding-right: calc(#{$quantity-selector-btn-padding-x} - var(--#{$prefix}border-width)); // stylelint-disable-line function-disallowed-list
```

Comment:
Don't know what to think about this solution. It works and the square looks like a perfect square even on focus but it introduces a small 2px shift on focus.

Reply:
TBH the rendering is weird as we can see the "plus" moving.